crates/db_schema/src/aggregates/site_aggregates.rs

```diff
@@ -3,13 +3,18 @@ use crate::{
   schema::site_aggregates,
   utils::{get_conn, DbPool},
 };
-use diesel::result::Error;
+use diesel::{result::Error, ExpressionMethods, QueryDsl};
 use diesel_async::RunQueryDsl;
 
 impl SiteAggregates {
+  /// reads the site aggregate for site_id = 1 (the local site)
   pub async fn read(pool: &mut DbPool<'_>) -> Result<Self, Error> {
+    use site_aggregates::dsl::site_id;
     let conn = &mut get_conn(pool).await?;
-    site_aggregates::table.first::<Self>(conn).await
+    site_aggregates::table
+      .filter(site_id.eq(1))
```

Comment:
Rather than doing all this with `id = 1`, the migration below should just delete site_aggregates for non-local sites, and some testing should be done to see if non-local site_aggregate rows are wrongly being inserted due to faulty triggers.